README.md

```diff
@@ -31,10 +42,30 @@ npm install react-sticky
 * `calculatedHeight` _(number)_ - height of the element returned by this function
 * `style` _(object)_ - modifiable style attributes to optionally be passed to the element returned by this function
 
+So, let's add to our example above. Inside of your `<Sticky />` tag, wrap the child element in a function and pass the relevant properties to the child.
+
+```js
+<StickyContainer>
+  <Sticky>
+    {({ style }) => {
+      return (
+        <h1 style={style}>
+          {"I'm Sticky"}
+        </h1>
+      )
+    }}
+  </Sticky>
+</StickyContainer>
+```
```

Comment:
I much prefer dropping `{}` from arrow functions that only return. Also, don't need `{" "}` around plain ol strings.


```
<StickyContainer>
  <Sticky>
    {({ style }) =>
      <h1 style={style}>I'm sticky</h1>
    }
  </Sticky>
</StickyContainer>
```

Reply:
OK, that's fine by me!